test/cmdLineTests/jvmtitests/src/com/ibm/jvmti/tests/nestMatesRedefinition/nmr001.java

```diff
@@ -102,7 +82,7 @@ public boolean testRedefinitionWithNestMembersAttribute() {
 			byte[] bytes = ClassGenerator.nestMemberWithNestHostAttribute();
 			Class<?> clazz = classloader.getClass("NestMembers", bytes);
 			Object instance = clazz.getDeclaredConstructor().newInstance();
-			bytes = ClassGenerator.nestMemberNoAttributeForceRedefinition();
+			bytes = ClassGenerator.nestMemberWithNestHostAttributeForceRedefinition();
```

Comment:
Was the test failing?  If it wasn't, then we have a bug else where and keeping the equivalent of the original test is very important.